lib/Core/Site/Values/Fields.php

```diff
@@ -236,6 +236,45 @@ public function getFieldById($id): APIField
         return $this->getSurrogateField((string) $id, $this->content);
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     */
+    public function getFirstSetField(string $firstIdentifier, string ...$otherIdentifiers): APIField
+    {
+        $identifiers = \array_merge([$firstIdentifier], $otherIdentifiers);
+        $fields = $this->getAvailableFields($identifiers);
+
+        foreach ($fields as $field) {
+            if (!$field->isEmpty()) {
```

Comment:
If we're checking for emptiness, should be call the method `getFirstNonEmptyField` or something similar?

"first set" field could be interpreted as the first field that exists in the content object.

Reply:
Makes sense, I'll update.

Reply:
Changed in https://github.com/netgen/ezplatform-site-api/pull/150/commits/f54d285ef778df22632e76347106d1c49d361b45